db/seeds.rb

```diff
@@ -6,11 +6,33 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-peter, katie = User.create!([
-  { name: "Peter", email: "peter@example.com", password: "password" },
-  { name: "Katie", email: "katie@example.com", password: "password2" },
-  ])
+interests = Faker::Lorem.words(20).map {|name| Interest.where({name: name}).first_or_create}
+skills = Faker::Lorem.words(20).map {|name| Skill.where({name: name}).first_or_create}
+professions = Faker::Lorem.words(20).map {|name| Profession.where({name: name}).first_or_create}
 
-peter.skills.create! ([
-  { name: "Ruby on Rails" },
-  ])
+50.times do
+  User.create!({
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    password: Faker::Internet.password,
+    interests: interests.sample(3),
+    skills: skills.sample(2),
+    professions: professions.sample(1),
+  })
+end
+
+20.times do
```

Comment:
You need to create some non-profits before doing this, and then link each event to a random non-profit.